server/models/Notification.js

```diff
@@ -0,0 +1,47 @@
+const mongoose = require('mongoose');
+
+const notificationSchema = new mongoose.Schema({
+  title: {
+    type: Number,
+    required: true,
+  },
+  message: {
+    type: String,
+    required: true,
+  },
+  receiver: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: 'users',
+  },
+  product: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: 'products',
+  },
+  old_price: {
+    type: Number,
+    required: true,
+  },
+  new_price: {
+    type: Number,
+    required: true,
+  },
+  read: {
+    type: Boolean,
+    required: true,
+    default: false,
+  },
```

Comment:
if we have different types of notifications (other than product price dropping), then these should probably not be required

Reply:
got it. thank you @sina-jamshidi